devops/kubernetes/cluster/index.ts

```diff
@@ -0,0 +1,32 @@
+import * as awsx from '@pulumi/awsx'
+import * as eks from '@pulumi/eks'
+import * as k8s from '@pulumi/kubernetes'
+import * as pulumi from '@pulumi/pulumi'
+
+const awsConfig = new pulumi.Config('aws')
+const config = new pulumi.Config()
+
+export let kubeconfig: pulumi.Output<any>
+export let provider: k8s.Provider
+export let platform = config.require('platform')
+
+if (platform === 'minikube') {
+  provider = new k8s.Provider('local', {})
+} else if (platform === 'aws') {
```

Comment:
I think the following parameters should be configurable when deploying the cluster:

- number of availability zones
- the cluster name
- instanceType